### PR TITLE
Format more individual code mentions in the readme

### PR DESCRIPTION
**Pull request by ming13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ onScreen<FormScreen> {
 
 ##### ListViews/RecyclersViews
 
-Kakao offers an easy way to interact with your RecyclerViews and ListViews
+Kakao offers an easy way to interact with your `RecyclerView`s and `ListView`s
```
Comment (Vacxe) [inline README.md]: I think we can remove "s" on ends, cuz I look a bit dirty

File: README.md [hunk 2]
```diff
@@ -173,14 +173,14 @@ onScreen<RecyclerScreen> {
...
-* childWith
+* `childAt`
+* `firstChild`
+* `lastChild`
+* `childWith`
 
 ##### Custom KViews
 
-If you have custom Views in your tests and you want to create your own `KView`, we have KBaseView. Just extend 
+If you have custom Views in your tests and you want to create your own `KView`, we have `KBaseView`. Just extend
```
Comment (Vacxe) [inline README.md]: Agree with this

**Comment by Vacxe** [comment]
@ming13 @Unlimity I think we can remove all code marks from the bullet point list. Maybe it just my opinion but it looks a bit ugly in the readme. What do you think guys?

**Comment by ming13** [comment]
For me personally code-fenced blocks mean that it is actually a piece of code I can reference, not a named concept or idea.

**Pull request merged**